src/config.ts

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs';
+import {SBSConfig} from "./types/config.model";
+
+const isTestMode = process.env.npm_lifecycle_script === 'ts-node test.ts';
+const configFile = isTestMode ? 'test.json' : 'config.json';
+export const config: SBSConfig = JSON.parse(fs.readFileSync(configFile).toString('utf8'));
+
+addDefaults(config, {
+    port: 80,
+    behindProxy: "X-Forwarded-For",
+    db: "./databases/sponsorTimes.db",
+    privateDB: "./databases/private.db",
+    createDatabaseIfNotExist: true,
+    schemaFolder: "./databases",
+    dbSchema: "./databases/_sponsorTimes.db.sql",
+    privateDBSchema: "./databases/_private.db.sql",
+    readOnly: false,
+    webhooks: [],
+    categoryList: ["sponsor", "intro", "outro", "interaction", "selfpromo", "music_offtopic"],
+    maxNumberOfActiveWarnings: 3,
+    hoursAfterWarningExpires: 24,
+    adminUserID: "",
+    discordCompletelyIncorrectReportWebhookURL: "",
+    discordFirstTimeSubmissionsWebhookURL: "",
+    discordNeuralBlockRejectWebhookURL: "",
+    discordReportChannelWebhookURL: "",
+    getTopUsersCacheTimeMinutes: 0,
+    globalSalt: null,
+    mode: "",
+    neuralBlockURL: null,
+    proxySubmission: null,
+    rateLimit: {
+        vote: {
+            windowMs: 900000,
+            max: 20,
+            message: "Too many votes, please try again later",
+            statusCode: 429,
+        },
+        view: {
+            windowMs: 900000,
+            max: 20,
+            statusCode: 200,
+            message: "",  // TODO TYPESCRIPT whats the msg?
```

Comment:
does this need a message?

Reply:
Wouldn't hurt to have a default value https://github.com/nfriedly/express-rate-limit/blob/master/lib/express-rate-limit.js#L23